packages/deployments/contracts/contracts/messaging/connectors/optimism/OptimismHubConnector.sol

```diff
@@ -3,16 +3,18 @@ pragma solidity 0.8.17;
 
 import {IRootManager} from "../../interfaces/IRootManager.sol";
 import {OptimismAmb} from "../../interfaces/ambs/optimism/OptimismAmb.sol";
-import {IStateCommitmentChain, L2MessageInclusionProof} from "../../interfaces/ambs/optimism/IStateCommitmentChain.sol";
+import {IOptimismPortal, ProvenWithdrawal} from "../../interfaces/ambs/optimism/IOptimismPortal.sol";
+import {IL2OutputOracle} from "../../interfaces/ambs/optimism/IL2OutputOracle.sol";
```

Comment:
it would be better to import these from the `@eth-optimism/contracts-bedrock` package.

unfortunately, I tried that and it fails because their contracts compile with solidity `0.8.15` :(